shared/chat/index.stories.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+import channelMentionHud from './conversation/input-area/channel-mention-hud/index.stories'
```

Comment:
we now load hierarchies of stories so the root just loads up the root chat stories and chat loads its components